packages/legend-studio/src/stores/editor-state/element-editor-state/mapping/MappingEditorState.ts

```diff
@@ -1277,6 +1268,23 @@ export class MappingEditorState extends ElementEditorState {
                   revealed = true;
                 }
               } else {
+                const extraCompilationErrorRevealers =
```

Comment:
Thinking about it, I don't think we need this, we can simplify the check up there to `this.currentTabState instanceof InstanceSetImplementationState`